### PR TITLE
Fix typo in DaoAuthenticationProvider JavaDoc

### PR DESCRIPTION
**Pull request by rymuff**
Fix typo in JavaDoc of USER_NOT_FOUND_PASSWORD

Add misssing `{@link`

### CODE REVIEW

File: core/src/main/java/org/springframework/security/authentication/dao/DaoAuthenticationProvider.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004, 2005, 2006 Acegi Technology Pty Limited
+ * Copyright 2002-2020 the original author or authors.
```
Comment (jzheaux) [inline core/src/main/java/org/springframework/security/authentication/dao/DaoAuthenticationProvider.java:2]: Will you please leave the `Acegi` copyright headers as-is?

**Review by jzheaux — CHANGES_REQUESTED** [review]
Hi, @rymuff! Thanks for the PR. I've left a piece of feedback inline.

File: core/src/main/java/org/springframework/security/authentication/dao/DaoAuthenticationProvider.java (hunk 1 above)
Comment (rymuff) [reply core/src/main/java/org/springframework/security/authentication/dao/DaoAuthenticationProvider.java]: done

**Comment by rymuff** [comment]
Hi, @jzheaux. Thanks for the feedback and I resolved it.

**Comment by jzheaux** [comment]
Thanks, @rymuff! Will you please also squash your commits? Then I think we're ready to merge.

**Comment by rymuff** [comment]
@jzheaux Done!

**Comment by jzheaux** [comment]
@rymuff, it looks like the PR build is failing due to formatting. Thanks for sticking with this, I realize it feels like several steps when it is a small change.

If you still have time, will you please run the following on the code so that it adjusts the formatting to meet the project guidelines:

```
./gradle format
```

Then, please squash again. If you are out of time, let me know and I'll make the change myself.

**Comment by rymuff** [comment]
@jzheaux Would you please make the change? sorry for bothering you.

**Comment by jzheaux** [comment]
No problem, @rymuff. Thanks for the contribution and hope to see you contribute more!

**Pull request closed without merging**